### ISSUE TITLE
Tamagui starter fails yarn native Android build

### ISSUE THREAD

**Issue by KaivG** [tamagui/tamagui#1043]
### Current Behavior

With starter app: npm create tamagui@latest when running yarn native and opening Expo Go on Android I get the following error:

 

> ERROR  Invariant Violation: requireNativeComponent: "RNSVGSvgViewAndroid" was not found in the UIManager.
> 
> This error is located at:
>     in RNSVGSvgViewAndroid (created by Svg)
>     in Svg (created by ChevronUp)
>     in ChevronUp (created by wrapped)
>     in wrapped
>     in RCTView (created by View)
>     in View (created by Button)
>     in Button
>     in ButtonProvider
>     in Button
>     in _Theme
>     in WithTheme (created by SheetDemo)
>     in SheetDemo (created by HomeScreen)
>     in RCTView (created by View)
>     in View
>     in Unknown (created by HomeScreen)
>     in HomeScreen (created by Screen)
>     in Screen (created by Route(Screen))
>     in Route (created by Route(Screen))
>     in LocationProvider (created by Route(Screen))
>     in Route(Screen) (created by SceneView)
>     in StaticContainer
>     in EnsureSingleNavigator (created by SceneView)
>     in SceneView (created by SceneView)
>     in RCTView (created by View)
>     in View (created by DebugContainer)
>     in DebugContainer (created by MaybeNestedStack)
>     in MaybeNestedStack (created by SceneView)
>     in RCTView (created by View)
>     in View (created by SceneView)
>     in RNSScreen (created by AnimatedComponent)
>     in AnimatedComponent
>     in AnimatedComponentWrapper (created by InnerScreen)
>     in Suspender (created by Freeze)
>     in Suspense (created by Freeze)
>     in Freeze (created by DelayedFreeze)
>     in DelayedFreeze (created by InnerScreen)
>     in InnerScreen (created by Screen)
>     in Screen (created by SceneView)
>     in SceneView (created by NativeStackViewInner)
>     in Suspender (created by Freeze)
>     in Suspense (created by Freeze)
>     in Freeze (created by DelayedFreeze)
>     in DelayedFreeze (created by ScreenStack)
>     in RNSScreenStack (created by ScreenStack)
>     in ScreenStack (created by NativeStackViewInner)
>     in NativeStackViewInner (created by NativeStackView)
>     in RCTView (created by View)
>     in View (created by SafeAreaInsetsContext)
>     in SafeAreaProviderCompat (created by NativeStackView)
>     in NativeStackView (created by NativeStackNavigator)
>     in PreventRemoveProvider (created by NavigationContent)
>     in NavigationContent
>     in Unknown (created by NativeStackNavigator)
>     in NativeStackNavigator
>     in Unknown (created by HomeLayout)
>     in ThemeProvider (created by HomeLayout)
>     in _ToastImperativeProvider (created by ToastProvider)
>     in ToastProviderProvider (created by ToastProvider)
>     in ToastCollectionProviderProvider (created by ToastCollectionProvider)
>     in ToastCollectionProvider (created by ToastProvider)
>     in ToastProvider (created by Provider)
>     in PortalProviderComponent
>     in _Theme (created by _ThemeProvider)
>     in _ThemeProvider (created by _TamaguiProvider)
>     in _TamaguiProvider
>     in Unknown (created by Provider)
>     in Provider (created by HomeLayout)
>     in HomeLayout (created by Route(HomeLayout))
>     in Route (created by Route(HomeLayout))
>     in LocationProvider (created by Route(HomeLayout))
>     in Route(HomeLayout) (created by RootRoute)
>     in RootRoute (created by ContextNavigator)
>     in InitialRootStateProvider (created by ContextNavigator)
>     in EnsureSingleNavigator
>     in BaseNavigationContainer
>     in ThemeProvider
>     in NavigationContainerInner (created by NavigationContainer)
>     in NavigationContainer (created by ContextNavigator)
>     in RootRouteNodeProvider (created by ContextNavigator)
>     in ContextNavigator (created by ExpoRoot)
>     in RNCSafeAreaProvider (created by SafeAreaProvider)
>     in SafeAreaProvider (created by ExpoRoot)
>     in RNGestureHandlerRootView (created by GestureHandlerRootView)
>     in GestureHandlerRootView (created by GestureHandler)
>     in GestureHandler (created by ExpoRoot)
>     in ExpoRoot (created by App)
>     in App (created by withDevTools(App))
>     in withDevTools(App) (created by ErrorOverlay)
>     in ErrorToastContainer (created by ErrorOverlay)
>     in ErrorOverlay
>     in RCTView (created by View)
>     in View (created by AppContainer)
>     in RCTView (created by View)
>     in View (created by AppContainer)
>     in AppContainer
>     in main(RootComponent), js engine: hermes
> Error: ENOENT: no such file or directory, open 'C:\Users\kudo\01_Work\Repos\expo\expo\android\versioned-react-native\ReactAndroid\hermes-engine\.cxx\MinSizeRel\5a4n485h\arm64-v8a\lib\InternalBytecode\InternalBytecode.js'
>     at Object.openSync (node:fs:585:3)
>     at Object.readFileSync (node:fs:453:35)
>     at getCodeFrame (C:\projects\home24\apps\test-android\node_modules\metro\src\Server.js:1004:18)
>     at Server._symbolicate (C:\projects\home24\apps\test-android\node_modules\metro\src\Server.js:1073:22)
>     at runMicrotasks (<anonymous>)
>     at processTicksAndRejections (node:internal/process/task_queues:96:5)
>     at async Server._processRequest (C:\projects\home24\apps\test-android\node_modules\metro\src\Server.js:437:7) {
>   errno: -4058,
>   syscall: 'open',
>   code: 'ENOENT',
>   path: 'C:\\Users\\kudo\\01_Work\\Repos\\expo\\expo\\android\\versioned-react-native\\ReactAndroid\\hermes-engine\\.cxx\\MinSizeRel\\5a4n485h\\arm64-v8a\\lib\\InternalBytecode\\InternalBytecode.js'
> }
> Error: ENOENT: no such file or directory, open 'C:\Users\kudo\01_Work\Repos\expo\expo\android\versioned-react-native\ReactAndroid\hermes-engine\.cxx\MinSizeRel\5a4n485h\arm64-v8a\lib\InternalBytecode\InternalBytecode.js'
>     at Object.openSync (node:fs:585:3)
>     at Object.readFileSync (node:fs:453:35)
>     at getCodeFrame (C:\projects\home24\apps\test-android\node_modules\metro\src\Server.js:1004:18)
>     at Server._symbolicate (C:\projects\home24\apps\test-android\node_modules\metro\src\Server.js:1073:22)
>     at runMicrotasks (<anonymous>)
>     at processTicksAndRejections (node:internal/process/task_queues:96:5)
>     at async Server._processRequest (C:\projects\home24\apps\test-android\node_modules\metro\src\Server.js:437:7) {
>   errno: -4058,
>   syscall: 'open',
>   code: 'ENOENT',
>   path: 'C:\\Users\\kudo\\01_Work\\Repos\\expo\\expo\\android\\versioned-react-native\\ReactAndroid\\hermes-engine\\.cxx\\MinSizeRel\\5a4n485h\\arm64-v8a\\lib\\InternalBytecode\\InternalBytecode.js'
> }
> Error: ENOENT: no such file or directory, open 'C:\Users\kudo\01_Work\Repos\expo\expo\android\versioned-react-native\ReactAndroid\hermes-engine\.cxx\MinSizeRel\5a4n485h\arm64-v8a\lib\InternalBytecode\InternalBytecode.js'
>     at Object.openSync (node:fs:585:3)
>     at Object.readFileSync (node:fs:453:35)
>     at getCodeFrame (C:\projects\home24\apps\test-android\node_modules\metro\src\Server.js:1004:18)
>     at Server._symbolicate (C:\projects\home24\apps\test-android\node_modules\metro\src\Server.js:1073:22)
>     at runMicrotasks (<anonymous>)
>     at processTicksAndRejections (node:internal/process/task_queues:96:5)
>     at async Server._processRequest (C:\projects\home24\apps\test-android\node_modules\metro\src\Server.js:437:7) {
>   errno: -4058,
>   syscall: 'open',
>   code: 'ENOENT',
>   path: 'C:\\Users\\kudo\\01_Work\\Repos\\expo\\expo\\android\\versioned-react-native\\ReactAndroid\\hermes-engine\\.cxx\\MinSizeRel\\5a4n485h\\arm64-v8a\\lib\\InternalBytecode\\InternalBytecode.js'
> }
> Error: ENOENT: no such file or directory, open 'C:\Users\kudo\01_Work\Repos\expo\expo\android\versioned-react-native\ReactAndroid\hermes-engine\.cxx\MinSizeRel\5a4n485h\arm64-v8a\lib\InternalBytecode\InternalBytecode.js'
>     at Object.openSync (node:fs:585:3)
>     at Object.readFileSync (node:fs:453:35)
>     at getCodeFrame (C:\projects\home24\apps\test-android\node_modules\metro\src\Server.js:1004:18)
>     at Server._symbolicate (C:\projects\home24\apps\test-android\node_modules\metro\src\Server.js:1073:22)
>     at runMicrotasks (<anonymous>)
>     at processTicksAndRejections (node:internal/process/task_queues:96:5)
>     at async Server._processRequest (C:\projects\home24\apps\test-android\node_modules\metro\src\Server.js:437:7) {
>   errno: -4058,
>   syscall: 'open',
>   code: 'ENOENT',
>   path: 'C:\\Users\\kudo\\01_Work\\Repos\\expo\\expo\\android\\versioned-react-native\\ReactAndroid\\hermes-engine\\.cxx\\MinSizeRel\\5a4n485h\\arm64-v8a\\lib\\InternalBytecode\\InternalBytecode.js'
> }

I also get a warning about incompatible dependencies with expo:
```

Some dependencies are incompatible with the installed expo version:
  expo-dev-client@2.1.6 - expected version: ~2.2.1
  expo-image@1.1.0 - expected version: ~1.0.0
  expo-splash-screen@0.18.1 - expected version: ~0.18.2
  expo-updates@0.16.3 - expected version: ~0.16.4
  react-native@0.71.5 - expected version: 0.71.7
  react-native-svg@13.9.0 - expected version: 13.4.0
Your project may not work correctly until you install the correct versions of the packages.
Install individual packages by running npx expo install expo-dev-client@~2.2.1 expo-image@~1.0.0 expo-splash-screen@~0.18.2 expo-updates@~0.16.4 react-native@0.71.7 react-native-svg@13.4.0
```

But running the recommended command in the apps/expo and packages/app folders still gives the same error when running yarn native

### Tamagui Version

```markdown
v1.17.0
```

### Reproduction

```markdown
npm create tamagui@latest
yarn native
Press a to open Android (Expo Go connected via usb)
```

### System Info

```markdown
System:
    OS: Windows 10 10.0.22621
    CPU: (12) x64 Intel(R) Core(TM) i7-10750H CPU @ 2.60GHz
    Memory: 4.37 GB / 15.75 GB
  Binaries:
    Node: 16.15.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 3.5.0 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 9.5.0 - C:\Program Files\nodejs\npm.CMD
  npmPackages:
    @babel/runtime: ^7.18.9 => 7.21.0
    @manypkg/cli: ^0.19.1 => 0.19.2
    check-dependency-version-consistency: ^3.0.3 => 3.0.3
    eslint: ^8.21.0 => 8.37.0
    node-gyp: ^9.3.1 => 9.3.1
    prettier: ^2.7.1 => 2.8.7
    turbo: ^1.8.3 => 1.8.8
    typescript: ^4.7.4 => 4.9.5
```

**Comment by KaivG** [comment]
I was able to fix the issue in the starter, I will post the steps shortly and see if I can create a pull request

**Comment by KaivG** [comment]
So the error `ERROR Invariant Violation: requireNativeComponent: "RNSVGSvgViewAndroid" was not found in the UIManager.`
is fixed by editing package.json in apps/expo to the specific version "react-native-svg": "13.4.0" instead of "react-native-svg": "^13.4.0". The newer version is not compatible with expo.

Also I noticed to other packages gave a warning that they were not compatible with expo and should probably also be fixed although I did not encounter any errors in the starter because of them:

Some dependencies are incompatible with the installed expo version:
  expo-image@1.2.1 - expected version: ~1.0.0
  react-native-safe-area-context@4.5.1 - expected version: 4.5.0

I also needed to delete and regenerate the yarn.lock file as it had a resolution to react-native-svg 13.9.0:
```
 "react-native-svg@npm:13.4.0":
  version: 13.9.0
```

I am not comfortable enough with this repo to test it and submit a pull request. Hope these explanations help to fix it
